metadata.yaml

```diff
@@ -63,6 +63,9 @@ requires:
       Certificate and key files for the loki server.
   catalogue:
     interface: catalogue
+  tracing:
+    interface: tracing
+    limit: 1
```

Comment:
Are there any reasons why we're doing this?

Reply:
After discussing with Mateusz, we can probably drop the limit if we want, but let's merge it as-is for now.